forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/DeviceBindAuthenticators.kt

```diff
@@ -79,6 +56,15 @@ interface DeviceAuthenticator {
         //Do Nothing
     }
 
+    /**
+     * The JWS algorithm (alg) parameter.
+     * Header Parameter identifies the cryptographic algorithm
+     * used to secure the JWS.
+     */
+    fun getAlgorithm(): String {
```

Comment:
Allow developer to override the sign algorithm

Reply:
may be we need this change in iOS also ? we need to double check that . i am sure we use EC, will need to look can we support RSA also

Reply:
We did some research before, and iOS only supports EC if I remember correctly. If that is the case, it may not be necessary for iOS to override. I think the client may want to change the alg for performance improvement.